style.css

```diff
@@ -384,118 +322,295 @@ textarea {
 
 ::placeholder {
   padding-left: 10px;
+  font-family: inherit;
   color: #ababab;
 }
 
 #cards-desktop {
   display: none;
 }
 
-#header-desktop {
+.lang-lang-desktop {
   display: none;
 }
 
-.lang-lang-desktop {
+.nav__container button {
+  display: inline-block;
+  font-size: 1.8rem;
+  cursor: pointer;
+  color: #6070ff;
+  border: none;
+  background-color: white;
+}
+
+#close-menu-btn {
   display: none;
 }
 
-/* Desktop */
-@media screen and (min-width: 768px) {
-  header {
-    height: 8%;
-    border-bottom: 1px solid #ebebff;
-  }
+#mobile-nav {
+  margin-bottom: -32px;
+  color: #6070ff;
+}
 
-  .container {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    flex-direction: row;
-    max-width: 85%;
-    margin: 5px auto;
-    text-align: center;
-  }
+.nav__container {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+  background-color: white;
+  position: relative;
+}
 
-  .desktop-nav-hidden {
-    width: 23%;
-    height: 40px;
-    display: block;
-    padding: 5px 10px;
-  }
+.nav__menu {
+  right: 3%;
+  flex-direction: column;
+  gap: 1rem;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 80%;
+  background: rgba(26, 84, 231, 0.8);
+  display: none;
+  overflow: auto;
+}
+
+.nav__menu li a {
+  width: 100%;
+  display: grid;
+  z-index: 1;
+  text-decoration: none;
+  position: relative;
+  color: white;
+  font-weight: bold;
+  font-size: 20px;
+}
 
-  .openMenu {
-    display: inline-block;
-    background-color: white;
-    position: static;
-    height: 8%;
-    overflow: hidden;
-    width: 23%;
-    margin: 0;
-    backdrop-filter: blur(10px);
+/* Popup */
+.popup {
+  height: 80vh;
+}
+
+.popup .overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.4);
+  z-index: 1;
+  display: none;
+  overflow: auto;
+}
+
+.content ul {
+  margin-top: 15px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.popup .content {
+  position: absolute;
+  margin: 60% 0;
+  top: 50%;
+  left: 45%;
+  transform: translate(-50%, -50%) scale(0);
+  background: white;
+  width: 80%;
+  height: 100%;
+  z-index: 2;
+  text-align: justify;
+  padding: 20px;
+}
```

Comment:
Your pop-up window doesn't follow the template design on the desktop version.

Your design:

<img width="1440" alt="Screen Shot 2022-06-24 at 8 21 47" src="https://user-images.githubusercontent.com/82426682/175544648-ecfadcd9-9da0-4d38-9c32-bd32709bf867.png">

The template design:

<img width="482" alt="Screen Shot 2022-06-24 at 8 22 30" src="https://user-images.githubusercontent.com/82426682/175544772-27abb5ed-7e4b-4138-b258-340f9cebdfa8.png">

Please fix this.